src/ui/CategoryFacet/CategoryFacetSearch.ts

```diff
@@ -223,7 +223,7 @@ export class CategoryFacetSearch implements IFacetSearch {
     );
     item.el.dataset.path = categoryFacetValue.value;
 
-    const countLabel = l('ResultCount', this.getFormattedCount(categoryFacetValue.numberOfResults));
+    const countLabel = l('ResultCount', this.getFormattedCount(categoryFacetValue.numberOfResults), categoryFacetValue.numberOfResults);
```

Comment:
Although this localization string only takes a single parameter, it has `<pl></pl>` tags that are only removed or applied if a second numeric parameter is given.

Reply:
I'm guessing we could have a very simple UT for this, that checks the label is properly localized (in english only)

Reply:
Alright, I'll add a UT.